e2e_tests/config/config.go

```diff
@@ -45,7 +45,7 @@ var (
 	testProjectIDs         = flag.String("test_project_ids", "", "test project ids")
 
 	// OutDir is the out directory to use.
-	OutDir = flag.String("out_dir", "/tmp", "junit xml directory")
+	OutDir = flag.String("out_dir", "/tmp", "artifact directory")
```

Comment:
is temp the artifact directory? may be use the env variable set by prow

Reply:
That is how it works, prow passes in the flag